environment/metadata.go

```diff
@@ -79,16 +86,37 @@ func fillECSData(e *MetaData, data *MetaDataStrings) *MetaData {
 	return e
 }
 
+func fillEC2PluginTests(e *MetaData, data *MetaDataStrings) *MetaData {
+	if len(data.EC2PluginTests) == 0 {
+		log.Println("Testing all EC2 plugins")
+		return e
+	}
+
+	plugins := strings.Split(strings.ReplaceAll(data.EC2PluginTests, " ", ""), ",")
+	log.Printf("Executing subset of plugin tests: %v", plugins)
+	m := make(map[string]struct{}, len(plugins))
+	for _, p := range plugins {
+		m[strings.ToLower(p)] = struct{}{}
+	}
+	e.EC2PluginTests = m
+
+	return e
+}
+
 func RegisterEnvironmentMetaDataFlags(metaDataStrings *MetaDataStrings) *MetaDataStrings {
 	registerComputeType(metaDataStrings)
 	registerECSData(metaDataStrings)
+	registerPluginTestsToExecute(metaDataStrings)
 	return metaDataStrings
 }
 
 func GetEnvironmentMetaData(data *MetaDataStrings) *MetaData {
 	metaData := &(MetaData{})
 	metaData = fillComputeType(metaData, data)
 	metaData = fillECSData(metaData, data)
+	if metaData.ComputeType == computetype.EC2 {
```

Comment:
```suggestion
```
and in `fillEC2PluginTests` we would have this check (for consistency between the fill environment function)

Reply:
What do you mean by this?

Reply:
The `fillECSData` has the following check
```
func fillECSData(e *MetaData, data *MetaDataStrings) *MetaData {
	if e.ComputeType != computetype.ECS {
		return e
	}
```
Therefore, instead of having `if metaData.ComputeType == computetype.EC2 {` here, would it be better to include the check in
```
func fillEC2PluginTests(e *MetaData, data *MetaDataStrings) *MetaData {
	if metaData.ComputeType == computetype.EC2 {
		return e
	}
```

Reply:
I think it's more confusing to always call the function and have it sometimes not do anything.